### PR TITLE
debian: packaging instructions

### PR DESCRIPTION
**Pull request by samuelkarp**
Packaging instructions for Debian

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: debian/packaging-debian.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Packaging sources for Debian are contained in the
+[`debian` branch](https://github.com/awslabs/amazon-ecr-credential-helper/tree/amazonlinux).
+
+This branch is updated every time a release is prepared for Debian.
+
+To prepare a release, you can use the following workflow:
+
+* Merge the release tag into this branch
+* Delete any files that have been added to the
```
Comment (petderek) [inline debian/packaging-debian.md]: can we programmatically check that this is empty?

**Comment by samuelkarp** [comment]
"Prep for release" is the overall workflow where "merge & do stuff" is the first step and "build package locally for testing" is the second.  Should I make that more clear?

File: debian/packaging-debian.md (hunk 1 above)
Comment (samuelkarp) [reply debian/packaging-debian.md]: A good idea, but not going to do it as part of this PR.

**Comment by petderek** [comment]
Maybe replace the line with:

`### Prepare a Release`

**Comment by samuelkarp** [comment]
@petderek Done

**Pull request merged**